playbooks/tasks/portal-role-task-mongo-setup-docker-service.yml

```diff
@@ -25,6 +24,8 @@
   when: mgkey_stat_result.stat.isreg | default(False)
   register: mgkey_update_result
```

Comment:
Not relevant to this PR but in here we might want to ensure the proper ownership and access rights of `mgkey` - it needs to be owned by user with id 999, which corresponds to `mongo` in the official MongoDB docker container. It also needs to have its access rights set to 0400.

Reply:
probably worth a follow up.

Reply:
if we check the permissions, and they are wrong, we can skip all the other checks and move to resetting the server since with the wrong permissions, mongo won't start.